apps/gp2-frontend/src/shared/__tests__/api.test.ts

```diff
@@ -12,10 +12,10 @@ describe('getKeywords', () => {
     nock.cleanAll();
   });
   it('returns a successfully fetched keywords list', async () => {
-    const keywordsResponse: gp2Model.ListKeywordsResponse =
-      gp2Fixtures.createKeywordsResponse();
+    const keywordsResponse: gp2Model.ListTagsResponse =
```

Comment:
maybe we should also rename variables. in this case, tagsResponse. and also update the test name